fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rewrite/ScalarOperatorFunctions.java

```diff
@@ -389,6 +420,55 @@ public static ConstantOperator divideDecimal(ConstantOperator first, ConstantOpe
         return createDecimalConstant(first.getDecimal().divide(second.getDecimal()));
     }
 
+    @FEFunction(name = "int_divide", argTypes = {"TINYINT", "TINYINT"}, returnType = "TINYINT")
```

Comment:
This is necessary？

Reply:
Yes.Such as "select 2 int_divide 1", the common type will be TINYINT, so the returnType is TINYINT too.